src/schedule/ll_schedule.c

```diff
@@ -223,7 +223,7 @@ static void schedule_ll_task_insert(struct task *task, struct list_item *tasks)
 	/* tasks are added into the list in order */
```

Comment:
@tlauda thanks for the patch. Please describe the initial problem in the commit message. What does 'right' order means.

Reply:
Just by looking at code I understand it as - previously task of the same priority that was added later would be triggered earlier, now they it should be intuitive so task are 'sorted by' priority and then order in which they were added.
@tlauda is it right?

Reply:
@jajanusz i think you are right but it is not trivial to understand this. Commits should always try to educate the reader and to clarify things.

A commit message should really state: WHY are we making this change. In particular in this case what is the problem and how it was solved:

So, instead of:

``
From now on tasks will be inserted in the right order based
on their priority and time of insertion

``
commit should read:

``There is a problem with the insertion of a task in the list of active(?) tasks. The list should
be sorted by task priority first and then when two tasks have the same priority we should look
of time of insertion. Newer tasks should be placed after older tasks in the list. Anyhow, in the
current implementaion newer task are placed before older tasks thus scheduled first. 

Fix this"